catkin_tools/argument_parsing.py

```diff
@@ -89,15 +89,15 @@ def add_cmake_and_make_and_catkin_make_args(parser):
 
     add = parser.add_mutually_exclusive_group().add_argument
     add('--make-args', metavar='ARG', dest='make_args', nargs='+', required=False, type=str, default=None,
-        help='Arbitrary arguments which are passes to make.'
+        help='Arbitrary arguments which are passes to make. '
```

Comment:
The string gets concatenated with the string immediately below. Without the change, you get:

```
$ catkin build -h
[ ... ]
  --make-args ARG [ARG ...]
                        Arbitrary arguments which are passes to make.It
                        collects all of following arguments until a "--" is
                        read.
```

Will fix passes -> passed as well.